framework/src/st_things/st_things.c

```diff
@@ -431,6 +480,11 @@ int st_things_register_reset_cb(st_things_reset_confirm_cb confirm_cb, st_things
 	return ST_THINGS_ERROR_NONE;
 }
 
+//--------------------------------------------------------------------------------
+/**
+ * Function Definition for st things reset.It call the things reset api,which delete
```

Comment:
1. We got a request to unify a word from SE team.
    st things? stthings? smartthings?
    @JeongJunSik @Hyunjun85  Could you confirm what is right?
2. It calls

Reply:
@harish-191 Could you please type "ST Things".
We have decided using "ST Things".

Reply:
ok